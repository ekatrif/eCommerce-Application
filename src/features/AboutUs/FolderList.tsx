import * as React from 'react';
import List from '@mui/material/List';
import ListItem from '@mui/material/ListItem';
import ListItemText from '@mui/material/ListItemText';
import ListItemAvatar from '@mui/material/ListItemAvatar';
import Avatar from '@mui/material/Avatar';
import CelebrationIcon from '@mui/icons-material/Celebration';
import Diversity3Icon from '@mui/icons-material/Diversity3';
import PermPhoneMsgIcon from '@mui/icons-material/PermPhoneMsg';
import TelegramIcon from '@mui/icons-material/Telegram';
import EditNoteIcon from '@mui/icons-material/EditNote';
import AddTaskIcon from '@mui/icons-material/AddTask';

export default function FolderList(): JSX.Element {
  return (
    <List
      sx={{
        width: '100%',
        bgcolor: 'background.paper',
        display: 'flex',
        flexWrap: 'wrap',
        justifyContent: 'space-around',
        rowGap: '15px',
        columnGap: '15px',
      }}
    >
      <ListItem sx={{ width: '49%' }}>
        <ListItemAvatar>
          <Avatar>
            <PermPhoneMsgIcon />
          </Avatar>
        </ListItemAvatar>
        <ListItemText primary="Weekly calls to discord" />
      </ListItem>
      <ListItem sx={{ width: '49%' }}>
        <ListItemAvatar>
          <Avatar>
            <EditNoteIcon />
          </Avatar>
        </ListItemAvatar>
        <ListItemText primary="Daily chat reports on the progress of tasks" />
      </ListItem>
      <ListItem sx={{ width: '49%' }}>
        <ListItemAvatar>
          <Avatar>
            <TelegramIcon />
          </Avatar>
        </ListItemAvatar>
        <ListItemText primary="Constant communication in telegram" />
      </ListItem>
      <ListItem sx={{ width: '49%' }}>
        <ListItemAvatar>
          <Avatar>
            <AddTaskIcon />
          </Avatar>
        </ListItemAvatar>
        <ListItemText primary="Using the task board in Trello" />
      </ListItem>
      <ListItem sx={{ width: '49%' }}>
        <ListItemAvatar>
          <Avatar>
            <Diversity3Icon />
          </Avatar>
        </ListItemAvatar>
        <ListItemText primary="Regular code review by mentor" />
      </ListItem>
      <ListItem sx={{ width: '49%' }}>
        <ListItemAvatar>
          <Avatar>
            <CelebrationIcon />
          </Avatar>
        </ListItemAvatar>
        <ListItemText primary="Gifs in pull requests" />
      </ListItem>
    </List>
  );
}
