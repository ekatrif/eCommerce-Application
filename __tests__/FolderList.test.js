import React from 'react';
import renderer from 'react-test-renderer';
import FolderList from '../src/features/AboutUs/FolderList';

test('FolderList renders correctly', () => {
  const tree = renderer
    .create(<FolderList />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
