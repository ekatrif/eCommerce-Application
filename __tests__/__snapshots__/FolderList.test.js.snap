// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FolderList renders correctly 1`] = `
<ul
  className="MuiList-root MuiList-padding css-1cq5zt0-MuiList-root"
>
  <li
    className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-ovffbk-MuiListItem-root"
    disabled={false}
  >
    <div
      className="MuiListItemAvatar-root css-1e9lk82-MuiListItemAvatar-root"
    >
      <div
        className="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-2s90m6-MuiAvatar-root"
      >
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="PermPhoneMsgIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M20 15.5c-1.25 0-2.45-.2-3.57-.57-.35-.11-.74-.03-1.02.24l-2.2 2.2c-2.83-1.44-5.15-3.75-6.59-6.58l2.2-2.21c.28-.27.36-.66.25-1.01C8.7 6.45 8.5 5.25 8.5 4c0-.55-.45-1-1-1H4c-.55 0-1 .45-1 1 0 9.39 7.61 17 17 17 .55 0 1-.45 1-1v-3.5c0-.55-.45-1-1-1zM12 3v10l3-3h6V3h-9z"
          />
        </svg>
      </div>
    </div>
    <div
      className="MuiListItemText-root css-tlelie-MuiListItemText-root"
    >
      <span
        className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
      >
        Weekly calls to discord
      </span>
    </div>
  </li>
  <li
    className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-ovffbk-MuiListItem-root"
    disabled={false}
  >
    <div
      className="MuiListItemAvatar-root css-1e9lk82-MuiListItemAvatar-root"
    >
      <div
        className="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-2s90m6-MuiAvatar-root"
      >
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="EditNoteIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M3 10h11v2H3v-2zm0-2h11V6H3v2zm0 8h7v-2H3v2zm15.01-3.13.71-.71c.39-.39 1.02-.39 1.41 0l.71.71c.39.39.39 1.02 0 1.41l-.71.71-2.12-2.12zm-.71.71-5.3 5.3V21h2.12l5.3-5.3-2.12-2.12z"
          />
        </svg>
      </div>
    </div>
    <div
      className="MuiListItemText-root css-tlelie-MuiListItemText-root"
    >
      <span
        className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
      >
        Daily chat reports on the progress of tasks
      </span>
    </div>
  </li>
  <li
    className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-ovffbk-MuiListItem-root"
    disabled={false}
  >
    <div
      className="MuiListItemAvatar-root css-1e9lk82-MuiListItemAvatar-root"
    >
      <div
        className="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-2s90m6-MuiAvatar-root"
      >
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="TelegramIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M9.78 18.65l.28-4.23 7.68-6.92c.34-.31-.07-.46-.52-.19L7.74 13.3 3.64 12c-.88-.25-.89-.86.2-1.3l15.97-6.16c.73-.33 1.43.18 1.15 1.3l-2.72 12.81c-.19.91-.74 1.13-1.5.71L12.6 16.3l-1.99 1.93c-.23.23-.42.42-.83.42z"
          />
        </svg>
      </div>
    </div>
    <div
      className="MuiListItemText-root css-tlelie-MuiListItemText-root"
    >
      <span
        className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
      >
        Constant communication in telegram
      </span>
    </div>
  </li>
  <li
    className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-ovffbk-MuiListItem-root"
    disabled={false}
  >
    <div
      className="MuiListItemAvatar-root css-1e9lk82-MuiListItemAvatar-root"
    >
      <div
        className="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-2s90m6-MuiAvatar-root"
      >
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="AddTaskIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M22 5.18 10.59 16.6l-4.24-4.24 1.41-1.41 2.83 2.83 10-10L22 5.18zM12 20c-4.41 0-8-3.59-8-8s3.59-8 8-8c1.57 0 3.04.46 4.28 1.25l1.45-1.45C16.1 2.67 14.13 2 12 2 6.48 2 2 6.48 2 12s4.48 10 10 10c1.73 0 3.36-.44 4.78-1.22l-1.5-1.5c-1 .46-2.11.72-3.28.72zm7-5h-3v2h3v3h2v-3h3v-2h-3v-3h-2v3z"
          />
        </svg>
      </div>
    </div>
    <div
      className="MuiListItemText-root css-tlelie-MuiListItemText-root"
    >
      <span
        className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
      >
        Using the task board in Trello
      </span>
    </div>
  </li>
  <li
    className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-ovffbk-MuiListItem-root"
    disabled={false}
  >
    <div
      className="MuiListItemAvatar-root css-1e9lk82-MuiListItemAvatar-root"
    >
      <div
        className="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-2s90m6-MuiAvatar-root"
      >
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="Diversity3Icon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M6.32 13.01c.96.02 1.85.5 2.45 1.34C9.5 15.38 10.71 16 12 16s2.5-.62 3.23-1.66c.6-.84 1.49-1.32 2.45-1.34-.72-1.22-3.6-2-5.68-2-2.07 0-4.96.78-5.68 2.01zM4 13c1.66 0 3-1.34 3-3S5.66 7 4 7s-3 1.34-3 3 1.34 3 3 3zm16 0c1.66 0 3-1.34 3-3s-1.34-3-3-3-3 1.34-3 3 1.34 3 3 3zm-8-3c1.66 0 3-1.34 3-3s-1.34-3-3-3-3 1.34-3 3 1.34 3 3 3z"
          />
          <path
            d="M21 14h-3.27c-.77 0-1.35.45-1.68.92-.04.06-1.36 2.08-4.05 2.08-1.43 0-3.03-.64-4.05-2.08-.39-.55-1-.92-1.68-.92H3c-1.1 0-2 .9-2 2v4h7v-2.26c1.15.8 2.54 1.26 4 1.26s2.85-.46 4-1.26V20h7v-4c0-1.1-.9-2-2-2z"
          />
        </svg>
      </div>
    </div>
    <div
      className="MuiListItemText-root css-tlelie-MuiListItemText-root"
    >
      <span
        className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
      >
        Regular code review by mentor
      </span>
    </div>
  </li>
  <li
    className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-ovffbk-MuiListItem-root"
    disabled={false}
  >
    <div
      className="MuiListItemAvatar-root css-1e9lk82-MuiListItemAvatar-root"
    >
      <div
        className="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-2s90m6-MuiAvatar-root"
      >
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="CelebrationIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="m2 22 14-5-9-9zm12.53-9.47 5.59-5.59c.49-.49 1.28-.49 1.77 0l.59.59 1.06-1.06-.59-.59c-1.07-1.07-2.82-1.07-3.89 0l-5.59 5.59 1.06 1.06zm-4.47-5.65-.59.59 1.06 1.06.59-.59c1.07-1.07 1.07-2.82 0-3.89l-.59-.59-1.06 1.07.59.59c.48.48.48 1.28 0 1.76zm7 5-1.59 1.59 1.06 1.06 1.59-1.59c.49-.49 1.28-.49 1.77 0l1.61 1.61 1.06-1.06-1.61-1.61c-1.08-1.07-2.82-1.07-3.89 0zm-2-6-3.59 3.59 1.06 1.06 3.59-3.59c1.07-1.07 1.07-2.82 0-3.89l-1.59-1.59-1.06 1.06 1.59 1.59c.48.49.48 1.29 0 1.77z"
          />
        </svg>
      </div>
    </div>
    <div
      className="MuiListItemText-root css-tlelie-MuiListItemText-root"
    >
      <span
        className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
      >
        Gifs in pull requests
      </span>
    </div>
  </li>
</ul>
`;
